README.md

```diff
@@ -82,6 +72,30 @@ Replace `bedita/web:latest` with `be4web-local` (or other chosen name) to lanch
 
 ## Development
 
+With this command you will launch a cake server and a webpack proxy server all with one command
+
+```bash
+yarn start
+```
+
+## Custom Dev Environment
+
+If you feel you want to customize ports, vhosts or you have your own php server then you can use these commands
+
+### Cake side App
+
+If you don't have a php enabled web server (apache...) you can use cake builtin server
+
+```bash
+bin/cake server
```

Comment:
This is not the recommended way to launch web app, not even for development. It only makes sense if you don't have a webserver and want to have a quick look....  

If you are a developer you better use `apache` or `nginx` not cake server.  We should be more clear on that.